Close button: Adding CSS var (#1531)
Orange-OpenSource/Orange-Boosted-Bootstrap: site/content/docs/5.2/components/close-button.md

## Without specific class

> Close buttons can also be created without `.btn-close` to avoid importing too much CSS bundle.

Reviewer: ```suggestion
Close buttons can also be created without `.btn-close` to reduce the size of your CSS bundle.
```

</button>
{{< /example >}}

## CSS

<!-- Boosted mod: CSS var -->
### Variables

{{< added-in "5.2.1" >}}

As part of Boosted’s evolving CSS variables approach, close buttons now use local CSS variables on `.btn-close` for enhanced real-time customization. Values for the CSS variables are set via Sass, so Sass customization is still supported, too.

{{< scss-docs name="btn-close-css-vars" file="scss/_close.scss" >}}

Customization through CSS variables can be seen on the `.btn-close-white` modifier class where we override specific values without adding duplicate CSS selectors.

{{< scss-docs name="btn-close-white-css-vars" file="scss/_close.scss" >}}
<!-- End mod -->

### Sass variables

Variables for all close buttons:

{{< scss-docs name="close-variables" file="scss/_variables.scss" >}}

> <!-- Boosted mod -->

Reviewer: You could rather create a big "Boosted mod" between line 52 and 76.

> Variables for the [white close button](#dark-variant):

Reviewer: ```suggestion
Variables for the [dark close button](#dark-variant):
```

Class name is white but this is the dark close button.
